webpack.config.js

```diff
@@ -106,7 +112,25 @@ module.exports = options => {
           test: /\.css$/,
           include: [path.resolve(__dirname, "node_modules/")], // only include 3rd party libraries
           use: styleLoaders
+        },
+        {
+          // Straightforward bundle of storybook/**/*.scss
+          test: /\/storybook\/.*\.scss$/,
+          use: ["style-loader", "css-loader", "sass-loader"]
         }
+        // TODO Bundling pictures from storybook/components/* - doesn't work right now
```

Comment:
let's remove the file loader added dependency if right now it's not working and in use?

Reply:
Dependency removed 👍